releases.md

```diff
@@ -0,0 +1,38 @@
+# Proposed Revised Repo Model and Release Plan
+
+The Node.js TSC has recently been discussing and hammering out the details
+for a revised general release plan. The fundamental details of the plan are:
+
+<img src="https://camo.githubusercontent.com/675f052419c56a583c19644d4da30cc0ff4e02bb/68747470733a2f2f636c6475702e636f6d2f6c4d48746245334e72782d3330303078333030302e706e67"/>
+
+The `master` branch remains as the target of primary development. The majority of commits generally land there. The overall goal is to keep master as stable as possible, with periodic, numbered `*-next.x` releases being cut at fairly regular intervals.
+
+Approximately every six months, a new *stable branch* is cut off master with an immediate `vX.0.0` release cut off that stable. Immediately after, the *semver-major* is bumped in `master`. In other words, if `master` is currently on `v3.0.0-next.x`, a new `v3.0.0` stable branch is cut off `v3.0.0-next.x` and `master` is bumped to `v4.0.0-next.x`. Six months later, a new `v4.0.0` stable branch is cut off `master` then `master` is bumped to `v5.0.0-next.x`.
+
+As mentioned, there will be fairly regular `next.x` releases off `master`. Developers can use these releases but ought to expect things to break once in a while. Breaking changes *will* land in `master` so it *is* possible (and *likely*) that breaking changes will occur from one `next.x` release to another. In particular, tools such as `NAN` may not yet have been updated to support the changes within any particular `next.x` release, and it's *likely* that native modules will not work.
+
+There will be *two* stable branches cut off `master` each year, overlapping each other by some period of time. From these stable branches, there will be a regular cadence of stable *beta* releases cut. Developers can (and *should*) use these and can be assured that there will *be no breaking changes* outside of the rare breaking security fix. A stable is *guaranteed* not to have *semver-major* bumps. Tools such as `NAN` *should* support all active stable releases.
+
+Of the two stables cut each year, the second will transition into the LTS at it's six month mark. The releases cut off this LTS-bound stable are essentially the LTS *release candidates*. Tools such as `NAN` *must* support all active LTS releases.
```

Comment:
Like shouldn't to the point that it would probably force us to delay it because then no-one's stuff will be updated and everything will break for a couple of weeks and we'll have a mess agin. :s